signer/src/testing/transaction_coordinator.rs

```diff
@@ -137,6 +137,9 @@ where
         mut self,
         delay_to_process_new_blocks: Duration,
     ) {
+        let _ = tracing_subscriber::fmt()
+            .with_env_filter(tracing_subscriber::EnvFilter::from_default_env())
+            .try_init();
```

Comment:
We should remove this. The issue is that only one tracing logger can be instantiated, so if another test adds this we'll get an error in CI.

One alternative is to have a static type that globally instantiates a logging thingy. Alternatively, there is this `test_log` utility that was added recently (I haven't played with it much, it seems to only log errors).

Reply:
Done